unitTests/test_asfvideo.cpp

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
-
+#ifdef EXV_ENABLE_VIDEO
```

Comment:
[suggestion] Instead of adding preprocessor conditions inside the code, I would prefer to add this logic in the file `unitTests/CMakeLists.txt`. If the variable is defined we add these .cpp files to the SOURCES of `unit_tests` and otherwise we do not do it.

Reply:
Done here b41332f0b26cff1442296d20354cbe8c4fdb666e